### PR TITLE
Removing secret migration functionality

### PR DESCRIPTION
**Pull request by fabiocav**

### CODE REVIEW

File: src/WebJobs.Script/ScriptConstants.cs [hunk 1]
```diff
@@ -48,7 +48,6 @@ public static class ScriptConstants
         public const string LogCategoryHost = "Host";
         public const string LogCategoryFunction = "Function";
         public const string LogCategoryWorker = "Worker";
-        public const string LogCategoryMigration = "Host.Migration";
```
Comment (mathewc) [inline src/WebJobs.Script/ScriptConstants.cs]: There is other code to remove

Comment (fabiocav) [reply src/WebJobs.Script/ScriptConstants.cs]: Yeah, thought about removing the properties. I'll do that.

**Comment by fabiocav** [comment]
The sentinel is still used by the blob repository

**Comment by fabiocav** [comment]
Updated based on feedback. Also fixed a broken test.

**Pull request merged**